CHANGELOG.md

```diff
@@ -244,6 +242,19 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (grpc) [\#8926](https://github.com/cosmos/cosmos-sdk/pull/8926) The `tx` field in `SimulateRequest` has been deprecated, prefer to pass `tx_bytes` instead.
 * (sdk types) [\#9498](https://github.com/cosmos/cosmos-sdk/pull/9498) `clientContext.JSONCodec` will be removed in the next version. use `clientContext.Codec` instead.
 
+## [v0.42.7](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.42.7) - 2021-07-09
```

Comment:
seems like we missed a entry for the capability global fix?

Reply:
ah you're right. Could you create a PR on release/v0.42.x? I'll add the same entry here afterwards